pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -0,0 +1,38 @@
+package harness_runner
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/prow"
+)
+
+var _ = ginkgo.Describe("[Suite: harnessess] Test Harness", func() {
+	defer ginkgo.GinkgoRecover()
+	h := helper.New()
+
+	TimeoutInSeconds := float64(viper.GetFloat64(config.Tests.PollingTimeout))
+	ginkgo.It("should run until completion", func(ctx context.Context) {
+		log.Printf("Test harness timeout is %v", TimeoutInSeconds)
```

Comment:
prefer `GinkgoWriter` to stdlib logger so this doesn't get printed outside of the test context

Reply:
Changed to ginkgo.GinkgoWriter.Write()
Though I'm not seeing this change in other test.